lib/ansible.js

```diff
@@ -30,11 +30,12 @@ AbstractAnsibleCommand.prototype.exec = function(options) {
     processOptions.cwd = options.cwd;
   }
 
-  // Make Ansible output unbuffered so the stdout and stderr `data` events 
+  // Make Ansible output unbuffered so the stdout and stderr `data` events
   // pickup output up more regularly
   var unbuffered = (options.buffered) ? '' : 1;
   processOptions.env = process.env;
   _.extend( processOptions.env, { PYTHONUNBUFFERED: unbuffered } );
+  _.extend( processOptions.env, options.env || {} );
```

Comment:
_.extend can accept multiple objects as sources. Please fix the code to look like this:
`_.extend( processOptions.env, options.env, { PYTHONUNBUFFERED: unbuffered } );